src/ui/components/Dashboard/NavBar.jsx

```diff
@@ -8,11 +8,18 @@ import TextField from '@material-ui/core/TextField';
 import NotificationsNoneIcon from '@material-ui/icons/NotificationsNone';
 import Avatar from '@material-ui/core/Avatar';
 import MenuOpenIcon from '@material-ui/icons/MenuOpen';
+import Badge from '@material-ui/core/Badge';
+import PropTypes from 'prop-types';
 import Menu from './Menu';
-function NavBar() {
+function NavBar(props) {
   const [open, setOpen] = useState(false);
-
+  const { newNotifs, callback } = props;
   const classes = useStyles();
+
+  const setNotif = useCallback(() => {
+    callback('notifications');
+  });
```

Comment:
Since `callback` is always used. just to set the screen can the name be updated to `setNotificationContent` or something like that?